packages/core/src/helpers/calls.ts

```diff
@@ -114,12 +114,17 @@ export function getCallsForUpdate(requests: RawCall[], options?: RefreshOptions)
 }
 
 /**
- * Result of a multicall call.
+ * Result of a [useCall](./02-Hooks.mdx#usecall) query.
```

Comment:
Remember that this jsdoc is visible in IDE as well - use `{@link useCall}`
The problem is that we have to make it work both in IDE and in docs - it is handled in the docs generation script for hooks, check it out.